pkg/serializer/decode.go

```diff
@@ -178,6 +202,7 @@ func (d *decoder) decode(doc []byte, into runtime.Object, ct ContentType) (runti
 // 	a returned failed because of the strictness using k8s.io/apimachinery/pkg/runtime.IsStrictDecodingError.
 // opts.DecodeListElements is not applicable in this call.
 // opts.ConvertToHub is not applicable in this call.
+// opts.DecodeUnknown is not applicable in this call
```

Comment:
point out that if you wanted to decode into an unknown object although it's registered in the scheme, you can do a decodeinto an unknown struct pointer